wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -19,23 +19,23 @@
 {% fragment as panel_id %}{{ prefix }}-section{% endfragment %}
 {% fragment as heading_id %}{{ prefix }}-heading{% endfragment %}
 {% fragment as content_id %}{{ prefix }}-content{% endfragment %}
-<section class="w-panel {{ classname }}" id="{{ panel_id }}" {% if heading %}aria-labelledby="{{ heading_id }}"{% endif %} data-panel>
+<section class="w-panel {{ classname }}" id="{{ panel_id }}" {% if heading %}aria-labelledby="{{ heading_id }}"{% endif %} w-panel data-controller="w-panel" >
     {# If a panel has no heading nor header controls, we don’t want any of the associated UI. #}
     {% if heading or header_controls %}
-        <div class="w-panel__header">
-            <a class="w-panel__anchor w-panel__anchor--prefix" href="#{{ panel_id }}" data-panel-anchor aria-labelledby="{{ heading_id }}">
+        <div class="w-panel__header" >
+            <a class="w-panel__anchor w-panel__anchor--prefix" href="#{{ panel_id }}" w-panel-anchor aria-labelledby="{{ heading_id }}">
```

Comment:
```suggestion
            <a class="w-panel__anchor w-panel__anchor--prefix" href="#{{ panel_id }}" data-w-panel-target="anchor" aria-labelledby="{{ heading_id }}">
```

Please note that using an attribute that is like `w-foo` is not a valid attribute. It will not break anything but should be avoided and is not necessary for this work either. 

Instead make this a target